upup/pkg/fi/nodeup/nodetasks/mount_disk.go

```diff
@@ -42,7 +42,28 @@ type MountDiskTask struct {
 var _ fi.Task = &MountDiskTask{}
 
 func (s *MountDiskTask) String() string {
-	return fmt.Sprintf("Disk: %s", s.Name)
+	return fmt.Sprintf("MountDisk: %s %s->%s", s.Name, s.Device, s.Mountpoint)
+}
+
+var _ CreatesDir = &MountDiskTask{}
```

Comment:
We are doing this in protokube as well.  Should we file an issue to refactor this into pkg and have both protokube and nodeup use this code?

Reply:
It's a nice idea - we could start to use tasks in protokube.  But send a PR instead of opening an issue :-)